x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/data/LongVectorFixedBuilder.java

```diff
@@ -38,6 +40,12 @@ public LongVectorFixedBuilder appendLong(long value) {
         return this;
     }
```

Comment:
Is this one still used? I think we could change the interface and just never call this. It's always slower. If it's still used, let's wait for a followup.

Reply:
There are still a couple of places using it, but I can try to remove it.
My only concern it that it's the entry point of `Builder` interface, so I'd have to make it throw an `UnsupportedOperationException`, or change the hierarchy.

Let me try and see how it goes

Reply:
If there are users hold it until the next PR please! It's nice to have this one be "mostly generated code"